src/modules/blockchain/blockchain-module-manager.js

```diff
@@ -510,6 +510,17 @@ class BlockchainModuleManager extends BaseModuleManager {
     async getParanetId(blockchain, knowledgeAssetId) {
         return this.callImplementationFunction(blockchain, 'getParanetId', [knowledgeAssetId]);
     }
+
+    async isCuratedNode(blockchain, paranetId, identityId) {
```

Comment:
You also need to add this function into implementation of module, in for blockchain module it would be web3-service.js